composer.json

```diff
@@ -40,12 +41,18 @@
         "psr/http-message-implementation": "^1.0",
         "php-http/client-implementation": "^1.0",
         "jms/serializer-bundle": "^1.5 || ^2.0 || ^3.0",
-        "ext-json": "*"
+        "ext-json": "*",
+        "http-interop/http-factory-guzzle": "^1.2"
```

Comment:
```suggestion
```

1. Library doesn't use this dependency
2. You're introducing a breaking change without the need to do that
3. Some dependencies are to be provided by the implementing application
  See https://github.com/prgTW/healthchecks-bundle/blob/509b3fc79d3ffc7a9f5bd498762c7172ea7b3ebd/composer.json#L40-L41